windows_containers/index.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-{productwinc} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With a Red Hat subscription, you can get support for running Windows workloads in {product-title}. Windows instances deployed by the WMCO are configured with the containerd container runtime. For more information, see the xref:../windows_containers/windows-containers-release-notes-10-15-x.adoc#windows-containers-release-notes-10-15-x[release notes].
+{productwinc} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With a Red Hat subscription, you can get support for running Windows workloads in {product-title}. Windows instances deployed by the WMCO are configured with the containerd container runtime. For more information, see the xref:../windows_containers/windows-rn/windows-containers-release-notes-10-15-x.adoc#windows-containers-release-notes-10-15-x[release notes].
```

Comment:
**PEER REVIEW** The only new text is in Release notes for past releases of the [Windows Machine Config Operator](https://github.com/openshift/openshift-docs/pull/74594/files#diff-bd0ee2ff751377a7d27a0d7725cf0e8ce6587bf14d3939ba944aca7aeaed7769R9-R11). The other text has been reviewed previously. 
Just need a sanity check on the re-org.